index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js" defer></script>
+        <title>Food shop</title>
+    </head>
+    
+<!-- header -->
+    <header>             
+        <div id="img-background"></div>  
+        <h1 id="ord-txt">Ordering food</h1>
+        <img id="logo" src="logo.jpg" alt="food">
+        <h3 id="logo-name">Fire delight</h3>
+        <hr id="sep1">
+
+<!-- top menu -->
```

Comment:
I think it helps to align the comment with the same indentation as the rows underneath, so I can understand it refers to them & my eyes don't jump to the start of the file